### PR TITLE
Add optional vault exporter for Prometheus.

### PR DESCRIPTION
**Pull request by ghost**
Signed-off-by: Dimitri Graf <dgraf@protonmail.ch>

#### What this PR does / why we need it:
This PR adds the possibility to deploy a Vault exporter sidecar container by setting some parameters. 
This is needed if you want to monitor your Vault with Prometheus. 

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: incubator/vault/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: A Helm chart for Vault, a tool for managing secrets
 name: vault
-version: 0.14.7
+version: 0.15.7
```
Comment (Towmeykaw) [inline incubator/vault/Chart.yaml]: This should be 0.15.0

Comment (ghost) [reply incubator/vault/Chart.yaml]: Hey Towmeykaw. I've just pushed the commit that sets the version number to 0.15.0. 
Is there anything else?

Comment (Towmeykaw) [reply incubator/vault/Chart.yaml]: Looks fine to me. But I'm not a maintainer

**Comment by dimitrigraf** [comment]
Hi Guys
I'm still waiting for feedback on this one. Is there anything missing? Can I do anything to speed up the process?
Best regards
Dimitri

File: incubator/vault/values.yaml [hunk 2]
```diff
@@ -9,6 +9,13 @@ image:
   tag:
   pullPolicy: IfNotPresent
 
+vaultExporter:
+  enabled: false
+  # repository: grapeshot/vault_exporter
```
Comment (paulczar) [inline incubator/vault/values.yaml]: If these values are all commented out, won't it fail if somebody sets `valtExporter` to true?    if these are sensible working defaults then they should be left uncommented.

**Review by paulczar — CHANGES_REQUESTED** [review]

File: incubator/vault/values.yaml (hunk 2 above)
Comment (ghost) [reply incubator/vault/values.yaml]: Yes, you are right. I've just uncommented all the sensible defaults except the CA cert one.

**Pull request merged**